### DISCUSSION TITLE
Please provide a portable win exe without installer, and a Linux binary

### DISCUSSION THREAD

**Discussion by raven-intheclouds** [Shubhamai/StableView#67] [Ideas]
Hi!

Great project, looking forward to having it working!

Right now I'm trying it on Windows via Wine, yet even if it shows the webcam output, it still says there's no device (see https://github.com/Shubhamai/StableView/issues/66).

1. For Wine and Lutris usage, it would be helpful if one had a portable .exe. That way, a folder with it can be shared via several Wine Prefixes (Wine windows filetree). This would mean it can be easily shared between for different games, instead of needing to install it once on each game.

2. Given that it communicates with Opentrack via UDP, it would also be nice to support building against Linux, and even providing a Linux binary.

I would be quite interested in (2).

Many thanks and cheers!

**Comment by Shubhamai** [comment]
1. Added in latest [pre-release](https://github.com/Shubhamai/StableView/releases/tag/v1.1.0).
2. Working on both Mac and Linux support, although it is available in latest [pre-release](https://github.com/Shubhamai/StableView/releases/tag/v1.1.0), it will likely not work due to libraries missing, working on resolving it.
